megamek/testresources/megamek/common/units/Seydlitz C.blk

```diff
@@ -0,0 +1,113 @@
+#building block data file
+<BlockVersion>
+1
+</BlockVersion>
+
```

Comment:
I think @rjhancock will agree with me that adding (duplicating) canon data in the tests may not be the best option given how we want to manage datafiles on the project, so I suggest that you either do the thing where it loads the file it's own data, or instead you add unnoficial units built exclusively for tests

Reply:
Actually, data for tests should be isolated from everything else and placed IN the test resources location SPECIFICALLY to deal with test isolation and ensure we can test good and bad files without impacting live data.